packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -13,193 +13,59 @@ import {IOracle} from "../interfaces/IOracle.sol";
 import {IController} from "../interfaces/IController.sol";
 import {IUniswapV3Pool} from "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
 import {IERC721Receiver} from "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
-import {ISwapRouter} from "@uniswap/v3-periphery/contracts/interfaces/ISwapRouter.sol";
 import {INonfungiblePositionManager} from "@uniswap/v3-periphery/contracts/interfaces/INonfungiblePositionManager.sol";
-import {IERC20Detailed} from "../interfaces/IERC20Detailed.sol";
 
 // contract
-// import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
 import {UniswapControllerHelper} from "./UniswapControllerHelper.sol";
 import {AaveControllerHelper} from "./AaveControllerHelper.sol";
 
 // lib
 import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
 import {Address} from "@openzeppelin/contracts/utils/Address.sol";
-import {TransferHelper} from "@uniswap/v3-periphery/contracts/libraries/TransferHelper.sol";
+import {ControllerHelperDataType} from "./lib/ControllerHelperDataType.sol";
+import {ControllerHelperUtil} from "./lib/ControllerHelperUtil.sol";
 
 /**
  * Error code
  * E0: user not allowed to operate vault
  * E1: max ETH to pay for long wPowerPerp is less than amount available
  * E2: ETH sent is less than amount to use for minting short plus amount to use for LPing
  * E3: amount out less than min
- * E4: amont in greater tuhan max
+ * E4: amont in greater than max
```

Comment:
*amount

Reply:
I removed all of those messages for bytecode limit